src/content/overview/observability/_index.md

```diff
@@ -1,12 +1,40 @@
+
+- **Alerting**: To not just look at dashboards all day it is crucial to be able to get notified about significant events or issues that require immediate attention. By making it easy to set up and configure alerting rules across apps and environments, our Observability Platforms alerting helps your teams avoid wasting time on repetitive tasks, thereby enhancing their efficiency and allowing them to focus on what matters.
+
+One of the key benefits using Giant Swarm is that we provide a set of battle-tested and highly integrated observability tools that our own teams already use on a daily basis and will help you have a comprehensive view of your applications and infrastructure.
+
+## Cloud-Native technologies
+
+- **Mimir** is an open source, horizontally scalable, highly available, multi-tenant time series database for long-term storage for metrics and serves as central core component for storing and analyzing metrics on all our managed clusters. Think of it as Prometheus on Steroids. Like Prometheus it integrates seamlessly with Kubernetes and collects, but also stores metrics for a longer period of time as time series data, enabling powerful querying and alerting capabilities.
+
+- **Loki** is an open source, horizontally scalable, highly available, multi-tenant log aggregation system inspired by Prometheus. It is designed to be very efficient and cost effective and integrates all sorts of logs in any format from any source, covering everything from app and system events to audit logs. With LogQL Loki also offers an easy Query Language simplifying querying logs. Additionally the language facilitates the generation of metrics from log data, a powerful feature that goes well beyond log aggregation.
+
+- **Grafana** is a popular open-source visualization tool for exploring metrics and logs and creating dashboards and graphs for all observability needs. It perfectly integrates Mimir and Loki but also supports various other data sources, including Prometheus, Elasticsearch, InfluxDB, and others, making it versatile for visualizing all observability data in just one single place.
+
+- **Alertmanager** is part of the Grafana ecosystem and is responsible for handling all alerts. It manages the routing, grouping, and silencing of alerts, ensuring that the right people are notified at the right time. At the same time, it supports various notification methods, including email, Slack, and PagerDuty.
+
+- **Various Agents** like Grafana-Agent, Fluent-bit and others help the Observability Platform collect and integrate relevant data from Workload Clusters, making the process to add new data as simple as just setting up a new service monitor or add the right labels.
```

Comment:
```suggestion
- **Various Agents** like Grafana-Agent (now renamed Alloy), Fluent-bit and others help the Observability Platform collect and integrate relevant data from Workload Clusters, making the process to add new data as simple as just setting up a new service monitor or add the right labels.
```

Or do we want to go with the new app name already to avoid a rewrite in a month or two?